lib/tool_shed/webapp/frontend/src/components/pages/CitableRepositoryPage.vue

```diff
@@ -18,7 +16,14 @@ const error = ref<string | null>(null)
 
 async function update() {
     error.value = null
-    const { data } = await indexFetcher({ owner: props.username, name: props.repositoryName })
+    const { data } = await client.GET("/api/repositories", {
```

Comment:
We've lost all error handling here, no? The old middleware would have thrown on error.

Reply:
Good catch again :)
Fixing it...